modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/dataset_display/side_panel/SidePanel.js

```diff
@@ -422,13 +419,9 @@ export default class SidePanel extends React.Component {
 		);
 
 		return ReactDOM.createPortal(
-			this.props.spritemap ? (
-				<ClayIconSpriteContext.Provider value={this.props.spritemap}>
-					{content}
-				</ClayIconSpriteContext.Provider>
-			) : (
-				content
-			),
+			<ClayIconSpriteContext.Provider value={this.props.spritemap}>
+				{content}
+			</ClayIconSpriteContext.Provider>,
```

Comment:
AFAIK, you don't need this either, because the sprite context provided by the top-level `render` call will also flow down into the portal that you create here (that's one of the neatest thing about React portals; that the participate in the context hierarchy even if they are anchored at another point in the DOM hierarchy).

Reply:
That's true. Only reason I kept it was because I thought that was a Metal component needing it somewhere. Checked it again and I don't see it. Removing...